src/briefcase/commands/base.py

```diff
@@ -983,6 +982,8 @@ def _generate_template(self, template, branch, output_path, extra_context):
         except cookiecutter_exceptions.RepositoryCloneFailed as e:
             # Branch does not exist.
             raise TemplateUnsupportedVersion(branch) from e
+        except cookiecutter_exceptions.UndefinedVariableInTemplate as e:
+            raise BriefcaseConfigError(f"{e.message}") from e
```

Comment:
Since the exception isn't adding anything to the upstream message, this can be simplified:
```suggestion
            raise BriefcaseConfigError(e.message) from e
```